tests/test_layering.py

```diff
@@ -0,0 +1,74 @@
+from itertools import islice
+
+import ppb.camera as camera
+import ppb.scenes as scenes
+
+
+class LayeredSprite:
+
+    def __init__(self, layer):
+        self.layer = layer
+
+
+class NoLayer:
+    pass
+
+
+def test_layering_attribute():
+
+    class LayeredScene(scenes.BaseScene):
+
+        def __init__(self):
+            super().__init__()
+            for x in range(5):
+                self.add(LayeredSprite(x))
+
+    scene = LayeredScene()
+    for lower_sprite, higher_sprite in zip(scene.sprite_layers(), islice(scene.sprite_layers(), 1, None)):
```

Comment:
should `.sprite_layers()` include the camera at all?

Reply:
The primary point of sprite_layers is things that should be rendered, so. . . probably not? We can probably exclude both DoNotRender (and whatever follows it) and size 0 objects.

Reply:
Ok, should it be the scene's or the renderer's responsibility to manage render order and filter for renderability?

Reply:
From discord:

Layered iteration is a scene concern, renderability is a renderer concern.

Also, Camera should stop being a sprite. To be worked out in a different PR.